.github/workflows/update_contributors.yaml

```diff
@@ -10,18 +10,24 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - name: Use Node.js 14.x
+    - name: Use Node.js 16.x
       uses: actions/setup-node@v3
       with:
-        node-version: 14.x
-    - name: install contributors
+        node-version: 16.x
+    - uses: actions/checkout@master
+      with:
+        repository: sighphyre/svg-contributors
+        path: './svg-contributors'
+    - name: Create Contributors Image
       run: |
-        npx svg-contributors -o unleash -n unleash
+        npm install --prefix ./svg-contributors
+        ./svg-contributors/bin/contributors -o unleash -n unleash -l 500
     - uses: aws-actions/configure-aws-credentials@v1
       with:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
     - name: Publish static assets to S3
       run: |
-        aws s3 cp ./contributors.svg s3://getunleash-static/docs-assets/contributors.svg
\ No newline at end of file
+        ls -l
```

Comment:
Why are we listing files? Is this a debugging leftover or does it solve some weird (or not-so-weird) issue?

Reply:
Sorry I missed this comment! Yeah this is debugging info, you're at the mercy of the CDN to confirm the correct image was build, which is an hour long time out. This is super useful to check against a local image to confirm the correct image was built